benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -293,17 +293,15 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
               << '\n';
   }
 
+  if (partition_dim < 1 || partition_dim > 3)
```

Comment:
Any reason to move this?

Reply:
I think there was one, but can't recall it now and don't see why.